recipes/r-boot/build.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
-
-# R refuses to build packages that mark themselves as Priority: Recommended
-mv DESCRIPTION DESCRIPTION.old
-grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```

Comment:
That I know :smile: 

My question was more general, related to the actual use of that when adding R recipes. I have a bunch of recipes ready but I want to put together some guidelines first.